phys/module_physics_init.F

```diff
@@ -1517,6 +1525,11 @@ SUBROUTINE phy_init ( id, config_flags, DT, restart, zfull, zhalf,     &
                 adv_moist_cond,                                 &
                 MPDT, DT, DX, DY, LOWLYR,                       &
                 F_ICE_PHY,F_RAIN_PHY,F_RIMEF_PHY,               &
+!-------------------for ntu3m------------------------------------
+#if ( EM_CORE == 1 )
+                PRES,RHOA,XLAND,CCNTY,QDCN,QTCN,QCCN,QRCN,QNIN, &
```

Comment:
@tzuchin12 
Two lines to allow conventional "&" ending

Reply:
Modified as suggested.